game/highscores.txt

```diff
@@ -1,6 +1,6 @@
 Most goals in one match:1587
 Win with most score difference:167
 Most ball posession ever:100 %
-Longest game:179 seconds
-Fastest goal:179 seconds
-Fastest win:179 seconds
+Longest game:441 seconds
+Fastest goal:434 seconds
```

Comment:
Kan dit een bug zijn in de highscores? 179 sec is sneller dan 434 en toch update hij de score